addons/website/static/src/js/backend/res_config_settings.js

```diff
@@ -37,6 +37,11 @@ BaseSettingController.include({
      * @override
      */
     _onButtonClicked: function (ev) {
+        if (ev.data.attrs.name === 'action_ping_sitemap') {
+            // Stable fix for users who have a visible button.
+            alert(_t("Google doesn't need to be pinged anymore. It will automatically fetch your /sitemap.xml."));
```

Comment:
:eyes: 

Let's not be that violent and let's add an Odoo dialog, browser `alert` are really illegal.

![image](https://github.com/odoo/odoo/assets/30048408/55ce6a3b-5f84-400e-b3c3-f83b23b23a6c)